earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +47,21 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def working_dir = "../src/"
+
+// Check for either "-PRELEASE" or "-POPTIMIZE" command line parameters.
+// Return the correct directory based on which paramater is used.
+def native_x86 = {
+  if ( project.hasProperty( 'OPTIMIZE' ) ) { return "NATIVE-OPT-x86_64" }
+  if ( project.hasProperty( 'DEBUG' ) ) { return "NATIVE-DBG-x86_64" }
+  return "NATIVE-REL-x86_64"
+}  // end native_x86
+
+// Build the actual file to be used
+def gee_long_version_txt = working_dir + native_x86() + "/gee_long_version.txt"
```

Comment:
If you're using `scons release=1` to generate `gee_long_version.txt` below, you need to use `"../src/NATIVE-REL-x86_64/gee_long_version.txt"` as the path here.

- [ ] Make path to `gee_long_version.txt` consistent with the command to generate it.